homeassistant/components/worldclock/sensor.py

```diff
@@ -21,6 +21,7 @@
     {
         vol.Required(CONF_TIME_ZONE): cv.time_zone,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_TIME_FORMAT, default=TIME_STR_FORMAT): cv.datetime
```

Comment:
The validation requires to check the string literals and not datetime.

Reply:
So would I use cv.string or do we have any other validation used for %H %M type formats?

Reply:
Closing the loop. Ended up finding the right one.

Reply:
I doubt that `cv.time_period_str` will validate `"%H:%M"`...